inline-caching/README.md

```diff
@@ -0,0 +1,247 @@
+# Inline Cache with GemFire
+
+Using an inline cache with a database can help improve performance, reduce database scaling limits, and provide greater flexibility in terms of data retrieval. 
+It is an important tool for any application that relies heavily on database access, and can help ensure that critical data is always available when it is needed.
+
+GemFire's inline cache is a mechanism for temporarily storing frequently accessed data in memory, rather than querying a database every time the data is needed. 
+This can significantly improve the performance of database-driven applications, especially when dealing with large amounts of data or complex queries.
+
+One of the main reasons to use GemFire with a database is to address scaling limits of the database itself. 
+As the size and complexity of a database grows, it can become increasingly difficult to maintain fast and efficient access to the data. 
+This is especially true in situations where multiple users or applications are simultaneously accessing the database, leading to contention and slowdowns.
+
+By using GemFire's inline cache, frequently accessed data can be stored in GemFire's memory, reducing the number of queries that need to be made 
+to the database. This can help alleviate some of the scaling limits of the database, as it reduces the amount of load placed on the database server. 
+It can also help reduce network latency and improve overall system performance, as data can be retrieved locally rather than over a network connection.
+
+# Implementation
```

Comment:
I think this title and sentence can be deleted.

Reply:
removed